python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -349,6 +352,9 @@ class RandomForestRegressor(Base):
         self.n_streams = handle.getNumInternalStreams()
         self.seed = seed
 
+        if ((seed != -1) and (n_streams != 1)):
+            warnings.warn("Random seed only works with n_streams=1.")
```

Comment:
Given the inherent variance in the regressor, we should update the message in this warning to inform users that the seed doesn't fully reproduce the same results